### ISSUE TITLE
Examples/Samples?

### ISSUE THREAD

**Issue by keithnicholson** [darrenparkinson/SparkDotNet#27]
Any chance at adding a repo of examples of this being used with .NET?  I would specifically be interested in how this works with actually participating in video conferences.

**Comment by forderud** [comment]
Agree. I'm wondering about exactly the same myself. Need to know if this SDK will enable me to add video conferencing support to a Windows .Net application.

**Comment by darrenparkinson** [comment]
Hi. I don't believe it would.  It's simply a wrapper around the [webex api](https://developer.webex.com/docs) which I don't believe supports that.  The [browser sdk](https://developer.webex.com/docs/sdks/browser) does appear to support video due to the browser supporting audio/video.  These looking interesting: [getUserMedia](https://webex.github.io/webex-js-sdk/api/#mediagetusermedia) and [getMediaStreams](https://webex.github.io/webex-js-sdk/api/#meetinggetmediastreams)
